website/frontend/templates/docs/scripting.html

```diff
@@ -89,20 +144,31 @@ <h3 id="left_text_num">$left(text,num)</h3>
           </li>
         </ul>
 
+        <h3 id="right_text_num">$right(text,num)</h3>
+
+        <ul>
+          <li>Returns the last
```

Comment:
Nitpicking and I know it's not your fault, but either "Returns last" or make `$left` also "Returns the first".